ectoolkits/analysis/disdeg.py

```diff
@@ -0,0 +1,181 @@
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+from MDAnalysis.lib.distances import capped_distance
+from MDAnalysis.analysis.base import AnalysisBase
+
+
+
+def count_Oidxs_bonds(positions, o_idxs:list, conf_idxs, box, max_cutoff=1.4, min_cutoff=None, ):
```

Comment:
This function can be generalized and be reused to other classes and functions. Could you rename the function as `count_CN_AB`, 
the function argument `o_idxs` could be `A_idxs`, and `conf_idxs` could be `B_idx`.  
In addition, please use python typing package to complete the type hint, for example
```python
from typing import List
o_idxs: List[int]
```